resources/quality/zyyx_plus/ABS/zyyx_plus_abs_fast.inst.cfg

```diff
@@ -0,0 +1,23 @@
+[general]
+definition = zyyx_plus
+name = Fast ABS
+version = 4
+
+[metadata]
+material = generic_abs
+quality_type = fast
+setting_version = 3
```

Comment:
`setting_version` should be 23. If you are actually using old interpretation/naming of settings, they should be manually upgraded. Same for all other files.